### PR TITLE
integrate parallax effect into main page

### PR DESCRIPTION
**Pull request by MagoDelBlocco**
the (almost) final layout of the page

### CODE REVIEW

**Comment by MagoDelBlocco** [comment]
decided that the interests and background sections would be too small, so the interests section will get merged into background

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -28,25 +19,65 @@ <h1>My Portfolio</h1>
...
+        </p>
       </div>
 
-      <div id="funfacts" class="sections">
+      <header id="funfacts" class="sections">
         <h1 class="centered-text">Fun facts</h1>
+      </header>
+      <div class="main-text">
+        <ul>
+        <li>My favourite movie villain is Thanos <!-- TODO: insert quote thanos button here --></li>
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: indent the `<li>`-s by 2 spaces relative to the `<ul>` indentation.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -28,25 +19,65 @@ <h1>My Portfolio</h1>
...
+        </ul>
       </div>
 
-      <div id="contact" class="sections">
+      <header id="contact" class="sections">
         <h1 class="centered-text">Contact</h1>
+      </header>
+      <div class="main-text">
+        <p><a href="mailto:bgdn.ciobanu12@gmail.com">E-mail</a></p>
+        <p><a href="https://github.com/MagoDelBlocco">GitHub</p>
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: add `</a>` for consistency with line 79.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -44,26 +42,74 @@
...
+  font-size: 32px;
 }
 
 .visible {
   display:block;
 }
 
 #whoami {
-
+  top: 0px;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [omit unit specification after `0` values](https://google.github.io/styleguide/htmlcssguide.html#0_and_Units)

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -44,26 +42,74 @@
...
+.sections {
+  background-color: #777;
+  height: 70px;
+  position: -webkit-sticky;
+  position: sticky;
+  width: 100%;
+}
 
+.sections h1 {
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [use the shorthand `font` property](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties), here and below

**Review by krasimirgg — APPROVED** [review]
Looking good!

**Pull request merged**